ganga/GangaCore/GPIDev/Lib/Job/Job.py

```diff
@@ -219,6 +219,7 @@ class Job(GangaObject):
                                      'subjobs': ComponentItem('jobs', defvalue=GangaList(), sequence=1, protected=1, load_default=0, copyable=0, comparable=0, optional=1, proxy_get="_subjobs_proxy", doc='list of subjobs (if splitting)', summary_print='_subjobs_summary_print'),
                                      'master': ComponentItem('jobs', getter="_getMasterJob", transient=1, protected=1, load_default=0, defvalue=None, optional=1, copyable=0, comparable=0, doc='master job', visitable=0),
                                      'postprocessors': ComponentItem('postprocessor', defvalue=MultiPostProcessor(), doc='list of postprocessors to run after job has finished'),
+                                      'virtualization': ComponentItem('virtualization', defvalue=None, load_default=0, optional=1, doc='optional virtualization to be used'),
```

Comment:
Are there any adverse effects of this addition on existing jobs in a repository? If for example I had some jobs before upgrading to this Ganga version will they still load?

Reply:
yes, I checked it by first installing ganga from develop branch then I created few jobs. then I installed ganga from this branch and the old jobs are still compatible.

Reply:
ok excellent, thanks.